### PR TITLE
Update user guide

### LINKED ISSUE #178: [PE-D] List/ Taglist user guide explanation
A new user reading this guide will wonder, whats the use of list when its already listed by default in the UI? Perhaps you want to bring list feature after find feature in the documentation. Hope this helps

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Wincenttjoi/ped#5

### PR DESCRIPTION
**Pull request by khinkhinn**
- Added photos for tagadd, taskadd and taskdelete
- Reordered both contact and tag features such that it is now in this order "find, list, sort" (Closes #178)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -351,29 +359,26 @@ Want to remove a contact? Simply use our `delete` command with the index `INDEX`
...
 * Creates a specified tag without the need of a contact.
 * Tag will not be created if it already exists in the tag list.
 
 Examples:
-* `tagadd t/CS2103T` creates a tag named CS2103T without assigning to any contact
-
-#### Listing all tags: `taglist`
-
-Want to see all the tags you have added? Display them with simply the `taglist` command.
+* `tagadd t/person` creates a tag named personal without any contact being assigned to the tag.
```
Comment (yeo-kexin) [inline docs/UserGuide.md]: i think there's a slight typo here? not really sure if it's person or personal

**Review by yeo-kexin — COMMENTED** [review]
lgtm other than the comment!

**Comment by siqiang-ng** [comment]
might want to resolve the conflicts too!

File: docs/UserGuide.md (hunk 1 above)
Comment (khinkhinn) [reply docs/UserGuide.md]: oh yes thanks

**Review by yeo-kexin — APPROVED** [review]

**Pull request merged**